src/components/contact-us/contact-us-questions-validation.ts

```diff
@@ -148,6 +167,19 @@ export function validateContactUsQuestionsRequest(): ValidationChainFunc {
           { value, lng: req.i18n.lng }
         );
       }),
+    body("serviceTryingToUse")
```

Comment:
Just wondering, is this a duplicate of the above validation on line 161?

Reply:
Thanks @BeckaL  - I've just been testing this and you're correct. They're functionally equivalent. I'd understood the use of [optional](https://express-validator.github.io/docs/api/validation-chain/#optional) in the previous rule to mean that it was not a required field. Based on that misunderstanding, I'd introduced a new rule because the designs showed it as required. I've updated the commit to remove this new rule and pushed. Thanks for spotting this.